app/controllers/sessions_controller.rb

```diff
@@ -24,4 +22,15 @@ def destroy
   def session_params
     params.require(:session).permit(:email, :password, :remember_me)
   end
+
+  def check_if_activated(user)
```

Comment:
This method seems to be doing more than checking if the user is activated.  What about renaming the method to capture the logic and consequences?